src/js/disability-benefits/containers/StatusPage.jsx

```diff
@@ -40,10 +41,13 @@ class StatusPage extends React.Component {
       const phase = claim.attributes.phase;
       const showDecision = phase === FIRST_GATHERING_EVIDENCE_PHASE
         && !claim.attributes.waiverSubmitted;
+      const showDocsNeeded = !claim.attributes.decisionLetterSent &&
+        claim.attributes.documentsNeeded &&
+        itemsNeedingAttentionFromVet(claim.attributes.eventsTimeline) > 0;
 
       content = (
         <div>
-          {claim.attributes.documentsNeeded && !claim.attributes.decisionLetterSent
+          {showDocsNeeded
             ? <NeedFilesFromYou claimId={claim.id} events={claim.attributes.eventsTimeline}/>
```

Comment:
Could this component just take the number of files needed as a prop/argument instead of needing to filter the timeline again? Also, is there a corresponding spec we could add or modify to test that this component is only rendered when there are items needed from the vet?